tests/web/no_robo/default.properties.yml

```diff
@@ -0,0 +1 @@
+no_robo: 'no_robo'
```

Comment:
Correct. I'll fix it. It's a file that shouldn't be read (because no RoboFile.php) but I didn't test that is itsn't.